### PR TITLE
MT2MQ

### PR DESCRIPTION
**Pull request by mariecrane**
Added MT2MQ to tools folder, including XML and R files, .shed.yml, and a test data folder

### CODE REVIEW

File: tools/mt2mq/MT2MQ.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    <command detect_errors="exit_code"><![CDATA[
+
+#if $options.mode == "f" or $options.mode == "ft":
+    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' $options.ontology $mq_output
+#elif $options.mode == "t":
+    mkdir in_dir
+    #for $input in $options.input_files:
+        && cp '$input' 'in_dir/${input.name.rsplit('.',1)[0]}'
+    #end for
+    && Rscript '$__tool_directory__/MT2MQ.R' in_dir t NA $mq_output
```
Comment (bgruening) [inline tools/mt2mq/MT2MQ.xml:16]: $mq_output needs also to be quoted

File: tools/mt2mq/MT2MQ.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<tool id="mt2mq" name="MT2MQ" version="1.0">
+    <description>Tool to prepare metatranscriptomic outputs from ASaiM for Metaquantome</description>
+    <requirements>
+        <requirement type="package" version="1.2.1">r-tidyverse</requirement>
+        <requirement type="package" version="1.3.1">r-readr</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+
+#if $options.mode == "f" or $options.mode == "ft":
+    Rscript '$__tool_directory__/MT2MQ.R' '$options.input_files' '$options.mode' $options.ontology $mq_output
```
Comment (bgruening) [inline tools/mt2mq/MT2MQ.xml:10]: $mq_output needs to be quoted

**Comment by pravs3683** [comment]
@bgruening @jj-umn Any idea why "Combine chunked test result" is failing. I have tried fixing and modifying several things in the wrapper, but it didn' help.  Thanks!!

**Comment by bgruening** [comment]
The errors are here: https://github.com/galaxyproteomics/tools-galaxyp/pull/465/checks?check_run_id=790847966#step:12:685

**Comment by mariecrane** [comment]
@bgruening it should be ready now!

**Pull request merged**